src/hooks/useDeleteTodo.ts

```diff
@@ -0,0 +1,26 @@
+import { useContext, useState } from 'react';
+
+import { TodosContext } from '../context/TodoContext';
+
+export const useDeleteTodo = () => {
+  const [isDeleting, setDeliting] = useState(false);
```

Comment:
`setDeliting` => `setDeleting`

however, will be better to use `isLoading` in all cases when we wait for some response

Reply:
I thought about naming between isLoading and isDeleting,in first case we show loader,in two case operations name - delete😅